packages/client/components/Menu.tsx

```diff
@@ -167,9 +173,13 @@ const Menu = forwardRef((props: Props, ref: any) => {
         if (activeIdx !== null) {
           const itemHandle = itemHandles.current[activeIdx]
           itemHandle?.onClick?.(e)
+          if (itemHandle?.to) {
+            history.push(itemHandle?.to)
+          }
```

Comment:
Can you please elaborate on why this is needed, and why `to` is added to types elsewhere? Shouldn't `onClick` be enough?

Reply:
Added some context below.